modules/javafx.graphics/src/main/java/javafx/scene/SpotLight.java

```diff
@@ -45,18 +45,14 @@
  * The light cone is defined by 3 factors: an {@link #innerAngleProperty() inner angle}, an {@link #outerAngleProperty()
  * outer angle}, and a {@link #falloffProperty() falloff} factor. For a point whose angle to the light is {@code a}, if
  * {@code a < innerAngle} then that point receives maximum illumination, if {@code a > outerAngle} then that point
- * receives no illumination, and if {@code innerAngle < a < outerAngle} then the illumination is determined by the
+ * receives no illumination, and if {@code innerAngle <= a <= outerAngle} then the illumination is determined by the
  * formula
  * <pre>I = pow((cos(a) - cos(outer)) / (cos(inner) - cos(outer)), falloff)</pre>
  * which represents a drop in illumination from the inner angle to the outer angle. {@code falloff} determines the
- * behavior of the drop.
+ * behavior of the drop. The expect values are {@code 0 <= innerAngle <= outerAngle <= 180} and {@code falloff >= 0};
```

Comment:
That should be "expected" values, although since this is a specification, I would prefer to use "valid range". Perhaps something like: "The valid ranges for these properties are `{@code 0 <= innerAngle <= outerAngle <= 180}` and `{@code falloff >= 0}`; values outside either of these ranges can produce unexpected results".